assessment.js

```diff
@@ -32,6 +38,26 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.houses.update(
+  {name: "House Stark"},
+  {
+    people:["Ned Stark", "Arya Stark"]
+  }
+)
+
+db.houses.update(
+  {name:"House Targaryen"},
+  {
+    { $push:{ people:["Viserys Targaryen", "Daenerys Targaryen"] } }
+  }
+)
+
+db.houses.update(
+  {name:"House Arryn"},
+  {
+    { $push:{ people:["Jon Arryn"] } }
+  }
+)
 
```

Comment:
this doesn't run, be sure to use the docs for $push and possibly $each in this case (it's mentioned on the $push doc page).  Also closer reading, they're supposed to be in a "members" array.